public/assets/main.css

```diff
@@ -158,7 +159,12 @@ label.right {
   display: inline-block;
 }
 
-.inline { display: inline-block }
+.inline {
+  display: inline
```

Comment:
this seems like a dangerous change because all other inline elements were doing inline-block

Reply:
It seems, until you grep for "\.inline". It returned only assets/app/view/game/issue_shares.rb. There I changed it to .inline-block. All other inline-blocks are defined inline.